src/commands/common/validate-asset-command.js

```diff
@@ -96,13 +101,82 @@ class ValidateAssetCommand extends Command {
             }
         }
 
+        let paranetId;
+        if (storeType === LOCAL_STORE_TYPES.TRIPLE_PARANET) {
+            try {
+                const {
+                    blockchain: paranetBlockchain,
+                    contract: paranetContract,
+                    tokenId: paranetTokenId,
+                } = this.ualService.resolveUAL(paranetUAL);
```

Comment:
We should check if paranet exists here imo before doing anything

Reply:
Done